src/fuego/__init__.py

```diff
@@ -1,4 +1,6 @@
 from .run_on_spaces import github_run, run
 
 
+__all__ = ["github_run", "run"]
```

Comment:
I add it to avoid F401, since the imports on top are not used, you can either # noqa: F401 in the import line or just add `__all__`, whatever you prefer! :)

Reply:
Let's mark with noqa for now...I'll issue a PR after yours with lazy loading that'll resolve the underlying problem :) (same as we do in `huggingface_hub`/`transformers`, etc)

Reply:
actually fine if you leave it like this for now too since I'll just change it afterwards anyways